modules/api/sync/README.md

```diff
@@ -0,0 +1,13 @@
+# Sync
+
+## Getting Started
+
+```
+docker compose up -d
+docker compose run --rm app pip install -r requirements.txt
+docker compose run --rm app python index.py
+```
+
+```
+docker run --rm -it --volume `pwd`:/app --workdir /app jjrom/shp2pgsql -I -s 4326 -g geom zonesalerte.shp geozone > zonesalerte.sql
```

Comment:
Permet de transformer un fichier `.shp`  + `.shx` + `.dbf`  + `.prj` en `.sql`.

`docker run --rm -it --volume `pwd`:/app --workdir /app` -> lance un docker éphémère en en mettant les fichiers du dossier courant dans le dossier `/app` du docker.

`jjrom/shp2pgsql` -> l'image du docker qui contient le binaire shp2pgsql

`-I -s 4326 -g geom zonesalerte.shp geozone` arguments de la commande shp2pgsql. `-g` pour le nom de la colonne geometry, c'est la valeur par défaut donc en théorie inutile mais ça sert de documentation 🤷🏻‍♂️. `zonesalerte.shp` le chemin du fichier shape, il va automatiquement récupérer les autres fichiers  `.shx` + `.dbf`  + `.prj`. `geozone` -> le nom de la table.

`> zonesalerte.sql` -> le resultat de la commande dans un fichier `.sql`. Les 2 premières lignes sont à supprimer. Ça pourrait se faire directement dans la commande mais j'ai la flemme de chercher... donc c'est à faire manuellement pour le moment.

Début du fichier généré :
```
Shapefile type: Polygon
Postgis type: MULTIPOLYGON[2]
SET CLIENT_ENCODING TO UTF8;
SET STANDARD_CONFORMING_STRINGS TO ON;
BEGIN;
CREATE TABLE "geozone" (gid serial,
"id" varchar(32),
"dpt" varchar(3),
"type" varchar(3),
"libel" varchar(200));
ALTER TABLE "geozone" ADD PRIMARY KEY (gid);
SELECT AddGeometryColumn('','geozone','geom','4326','MULTIPOLYGON',2);
INSERT INTO "geozone" ("id","dpt","type","libel",geom) VALUES ('84_01_1','01','SUP','Saône Aval','0A5941...');
INSERT INTO "geozone" ("id","dpt","type","libel",geom) VALUES ('84_01_1','01','SOU','Saône Aval','A33932...');
```

On retrouve les 2 premieres lignes a supprimer.
La table et les champs de la table viennent du fichier `.dbf`. A une vache près c'est la même chose que ce qu'on avait prévu donc mieux vaut partir sur leur schema. Le dernier champ de l'insertion est tronqué.